presto-main/src/main/java/com/facebook/presto/cost/StatsCalculator.java

```diff
@@ -14,27 +14,24 @@
 package com.facebook.presto.cost;
 
 import com.facebook.presto.Session;
-import com.facebook.presto.spi.type.Type;
-import com.facebook.presto.sql.planner.Symbol;
+import com.facebook.presto.sql.planner.TypeProvider;
 import com.facebook.presto.sql.planner.iterative.IterativeOptimizer;
 import com.facebook.presto.sql.planner.iterative.Lookup;
 import com.facebook.presto.sql.planner.plan.PlanNode;
 
-import java.util.Map;
-
 public interface StatsCalculator
 {
     /**
      * Calculate stats for the {@code node}.
-     *
-     * @param node The node to compute stats for.
+     *  @param node The node to compute stats for.
      * @param sourceStats The stats provider for any child nodes' stats, if needed to compute stats for the {@code node}
      * @param lookup Lookup to be used when resolving source nodes, allowing stats calculation to work within {@link IterativeOptimizer}
+     * @param types
```

Comment:
not very useful, remove

Reply:
Heh... a side-effect of Intellij's method signature refactoring. Will do.